CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Program.cs

```diff
@@ -0,0 +1,136 @@
+        {
+            Logger.LogMessage(MessageType.INFORMATION, $"RecognizeCompleted event recieved for callconnetion id: {@event.CallConnectionId}");
+
+            var recognizeEvent = (RecognizeCompleted)@event;
+            var toneDetected = recognizeEvent.CollectTonesResult.Tones[0];
+            FileSource playSource;
+
+            if (toneDetected.Equals(DtmfTone.One))
+            {
+                playSource = new FileSource(new Uri(callConfiguration.AppBaseUri + Constants.AppointmentConfirmedAudio));
+            }
+            else if (toneDetected.Equals(DtmfTone.Two))
+            {
+                playSource = new FileSource(new Uri(callConfiguration.AppBaseUri + Constants.AppointmentCancelledAudio));
+            }
+            else // Invalid Dtmf tone
+            {
+                playSource = new FileSource(new Uri(callConfiguration.AppBaseUri + Constants.InvalidToneAudio));
+            }
+
+            // Play audio for dtmf response
+            await callAutomationClient.GetCallConnection(@event.CallConnectionId)
+                .GetCallMedia()
+                .PlayToAllAsync(playSource, new PlayOptions { OperationContext = "ResponseToDtmf", Loop = false });
+        }
+        if (@event is RecognizeFailed { OperationContext: "AppointmentReminderMenu" })
+        {
+            Logger.LogMessage(MessageType.INFORMATION, $"RecognizeFailed event recieved for callconnetion id: {@event.CallConnectionId}");
+
+            var playSource = new FileSource(new Uri(callConfiguration.AppBaseUri + Constants.InvalidToneAudio));
+            await callAutomationClient.GetCallConnection(@event.CallConnectionId)
+                .GetCallMedia()
+                .PlayToAllAsync(playSource);
+        }
+        if (@event is PlayCompleted { OperationContext: "ResponseToDtmf" })
+        {
+            Logger.LogMessage(MessageType.INFORMATION, $"PlayCompleted event recieved for callconnetion id: {@event.CallConnectionId}");
+
+            await callAutomationClient.GetCallConnection(@event.CallConnectionId).HangUpAsync(forEveryone: true);
+        }
+        if (@event is PlayFailed { OperationContext: "ResponseToDtmf" })
+        {
+            Logger.LogMessage(MessageType.INFORMATION, $"PlayFailed event recieved for callconnetion id: {@event.CallConnectionId}");
+
+            await callAutomationClient.GetCallConnection(@event.CallConnectionId).HangUpAsync(forEveryone: true);
+        }
+    }
+    return Results.Ok();
+}).Produces(StatusCodes.Status200OK);
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment() || app.Environment.IsProduction())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseStaticFiles(new StaticFileOptions
+{
+    FileProvider = new PhysicalFileProvider(
+           Path.Combine(builder.Environment.ContentRootPath, "audio")),
+    RequestPath = "/audio"
+});
+
+app.UseHttpsRedirection();
+
+app.UseAuthorization();
```

Comment:
are we using auth?

Reply:
No we are not, I will get rid of redundant code I forgot to remove them